app/static/js/grid_scripts.js

```diff
@@ -0,0 +1,48 @@
+const tolerance = 1;
+let last = 0;
+let lastIndex = 0;
+
+function setToggle(elementClass, targetElement, classToAdd) {
+    let allDays = document.getElementsByClassName(elementClass);
```

Comment:
const

Reply:
lastIndex is updating during the loop - from what I was reading the **const** is for variables that not going to change. 
so in this case onlt "allDays" can be const.